edtLHD/src/test/java/fr/univtln/lhd/model/entities/dao/users/StudentDAOTest.java

```diff
@@ -51,7 +51,7 @@ void addNewStudent() throws SQLException {
         dao.delete(student);
     }
 
-    //@Test Need to fix circular import
+    @Test //Need to fix circular import
```

Comment:
why keep the commentary ?

Reply:
forgot about it, i was just testing around and validating the test, i've forgot to remove that comment

Reply:
That's remove, juste waiting for @anastygnome approval to merge